### ISSUE TITLE
Check the IfW API

### ISSUE THREAD

**Issue by tbauriedel** [Icinga/icinga-powershell-framework#676]
To check the API of the IfW, you have to create a simple `Invoke-WebRequest` at the moment.

To make life easier here and have a builtin feature, it would be useful to have a command to check the API.
Something like `Test-IcingaForWindowsAPI` or something.

It should check whether the API is available and delivers a result.

Also it would be nice to have arguments to check the API from a remote host.
This would allow users to check whether the API of a remote host is accessible and is not blocked by any defined rules.

**Comment by LordHepipud** [comment]
#701
